app/controllers/api/v1/users/rents_controller.rb

```diff
@@ -1,17 +1,14 @@
 module Api
   module V1
     module Users
-      class RentsController < ApplicationController
-        before_action :authenticate_user!
-
-        include Wor::Paginate
-
+      class RentsController < ApiController
         def index
           render_paginated Rent.where(user_id: params[:user_id])
         end
 
         def create
           rent = Rent.create(rent_params)
+          RentNotificationsJob.perform_later(rent)
```

Comment:
It is better to just send the rent id here, that way you don't serialize the whole entity into redis, using potentially a lot of memory.